server/tests/backends/fixtures/evolution/perc_groups_mongo.json

```diff
@@ -0,0 +1,113 @@
+{
+  "exclude": [
+    "mysql",
+    "postgres",
+    "snowflake",
+    "pandas"
+  ],
+  "step": {
+    "pipeline": [
+      {
+        "name": "evolution",
+        "date_col": "DATE",
+        "value_col": "VALUE",
+        "evolution_type": "vsLastYear",
+        "evolution_format": "pct",
+        "index_columns": [
+          "COUNTRY"
+        ],
+        "new_column": "POUET"
+      }
+    ]
+  },
+  "input": {
+    "schema": {
+      "fields": [
+        {
+          "name": "DATE",
+          "type": "datetime"
+        },
+        {
+          "name": "VALUE",
+          "type": "number"
+        },
+        {
+          "name": "COUNTRY",
+          "type": "string"
+        }
+      ],
+      "pandas_version": "0.20.0"
+    },
+    "data": [
+      {
+        "DATE": "2019-01-01 00:00:01",
+        "VALUE": 2,
+        "COUNTRY": "France"
+      },
+      {
+        "DATE": "2019-01-01 00:00:02",
+        "VALUE": 1,
+        "COUNTRY": "Italy"
+      },
+      {
+        "DATE": "2020-01-01 00:00:01",
+        "VALUE": 4,
+        "COUNTRY": "France"
+      },
+      {
+        "DATE": "2020-01-01 00:00:02",
+        "VALUE": 0.5,
+        "COUNTRY": "Italy"
+      }
+    ]
+  },
+  "expected": {
+    "schema": {
+      "fields": [
+        {
+          "name": "DATE",
+          "type": "datetime"
+        },
+        {
+          "name": "VALUE",
+          "type": "number"
+        },
+        {
+          "name": "COUNTRY",
+          "type": "string"
+        },
+        {
+          "name": "POUET",
+          "type": "number"
+        }
+      ],
+      "pandas_version": "0.20.0"
+    },
+    "data": [
+      {
+        "DATE": "2019-01-01 00:00:01",
+        "VALUE": 2,
+        "COUNTRY": "France",
+        "POUET": null
+      },
+      {
+        "DATE": "2019-01-01 00:00:02",
+        "VALUE": 1,
+        "COUNTRY": "Italy",
+        "POUET": null
+      },
+      {
+        "DATE": "2020-01-01 00:00:01",
+        "VALUE": 4,
+        "COUNTRY": "France",
+        "POUET": null
+      },
+      {
+        "DATE": "2020-01-01 00:00:02",
+        "VALUE": 0.5,
+        "COUNTRY": "Italy",
+        "POUET": null
```

Comment:
This does not look right, have you been able to reproduce this behaviour with the typescript mongo translator ? Given the input, this should be the same as in `server/tests/backends/fixtures/evolution/perc_groups.json` . Can't spot a typo in the translator here though